tutorials/applications/Spin chain of 3 atoms in XY mode.ipynb

```diff
@@ -0,0 +1,406 @@
+{
```

Comment:
<p>This error comes from using <code>pulser</code> installed from PyPi instead of installed from source, but I'm sure you've solved it at this point. It would be good not to have it here though</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/198/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>